README.md

```diff
@@ -5,23 +5,23 @@ in the `cEP-0000.md` document.
-| [cEP-0014](cEP-0014.md) | Generate relevant coafile sections | This cEP proposes a framework for coala-quickstart to generate more relevant `.coafile` sections based on project files and user preferences.                                                                                                         |
-| [cEP-0018](cEP-0018.md) | Integration of ANTLR into coala    | This cEP describes how an API based on ANTLR will be constructed and maintained                                                                                                                                                                       |
-| [cEP-0019](cEP-0019.md) | Meta-review System                 | This cEP describes the details of the process of implementing the meta-review system as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).                                                              |
-| [cEP-0020](cEP-0020.md) | Newcomer Gamification system       | This cEP describes the details of the process of implementing the newcomer gamification system as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5892040252981248)                                                     |
-| [cEP-0021](cEP-0021.md) | Profile Bears                      | This cEP describes the detailed implementation of a profiling interface as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#6109762077327360).                                                                           |
-| [cEP-0022](cEP-0022.md) | coala-quickstart Green Mode        | This cEP provides the features, needs, requirements and implementation of the green-mode being added to coala-quickstart.                                                                                                                             |
-| [cEP-0025](cEP-0025.md) | Integrate pyflakes AST into coala  | This cEP describes the details of the process of integrating the pyflakes-enhanced-AST into coala as part of [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5549789140221952).                                                       |
-| [cEP-0027](cEP-0027.md) | coala Bears Testing API            | This cEP describes the implementation process of `BaseTestHelper` class and `GlobalBearTestHelper` class to improve testing API of coala as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#6625036551585792).          |
+| Filename                | Title                              | Date        | Description                                                                                                                                                                                                                                           |
```

Comment:
The name `Date` here is ambiguous. What date? Creation date? Last update date?

Reply:
`Date` refers Creation dates.

Reply:
updated.